examples/single_agent.py

```diff
@@ -1,68 +1,57 @@
 import logging
+import os
+import pathlib
 
 import gym
+import numpy as np
 
 from smarts.core.agent import Agent, AgentSpec
 from smarts.core.agent_interface import AgentInterface, AgentType
-from smarts.core.sensors import Observation
 from smarts.core.utils.episodes import episodes
 from smarts.env.wrappers.single_agent import SingleAgent
+from smarts.env.wrappers.standard_obs import StandardObs, StdObs
 
-# The following ugliness was made necessary because the `aiohttp` #
-# dependency has an "examples" module too.  (See PR #1120.)
```

Comment:
We can create an issue and link this ugliness inside the issue rather than adding ugliness to an especially user impression creating code.

Reply:
OK, point taken about the comment potentially adding to the existing ugliness.  

The reference to the _reason_ for the ugliness is what I don't want to lose (otherwise we'll be stuck with this ugliness forever, as opposed to hopefully cleaning it up once `aiohttp` is fixed.)...  I will create the issue to track this.

(fwiw, the reason I had the comment in the code instead of just in an issue was because this is in our public-facing examples area, and I didn't want potential users to "judge us" based upon this, when it wasn't really our fault that we had to make it so convoluted.)

Reply:
The latest version of `aiohttp` package does not cause issues with using the `example` module. The `argument_parser` imports have already been fixed.